adagrams/game.py

```diff
@@ -1,11 +1,85 @@
+import random 
+
+LETTERS_DICT = {'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2,\
+'I': 9, 'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, \
+'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1,'Y': 2, 'Z': 1}
+
+'''
+We generated LETTERS_POOL from LETTERS_DICT using the following code so that we
+would not have to run it every time draw_letters is called since the pool of 
+letters will be a constant
+LETTERS_POOL = []
+for letter in LETTERS_DICT:
+    letters_string = letter * LETTERS_DICT[letter]
+    for letter in letters_string:
+        LETTERS_POOL.append(letter)
+'''
+LETTERS_POOL = ['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'B', 'B', 'C', \
+'C', 'D', 'D', 'D', 'D', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', \
+'E', 'E', 'F', 'F', 'G', 'G', 'G', 'H', 'H', 'I', 'I', 'I', 'I', 'I', 'I', \
+'I','I', 'I', 'J', 'K', 'L', 'L', 'L', 'L', 'M', 'M', 'N', 'N', 'N', 'N', 'N', \
+'N', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'P', 'P', 'Q', 'R', 'R', 'R', 'R', \
+'R', 'R', 'S', 'S', 'S', 'S', 'T', 'T', 'T', 'T', 'T', 'T', 'U', 'U', 'U', 'U', \
+'V', 'V', 'W', 'W', 'X', 'Y', 'Y', 'Z']
```

Comment:
this works! Though I think since you put in the effort of coding the algorithm out, you should show it off! Maybe we can do something like this:

```suggestion
def get_letters_pool:
    for letter in LETTERS_DICT:
        letters_pool = []
        letters_string = letter * LETTERS_DICT[letter]
        for letter in letters_string:
            letters_pool.append(letter)
    return letters_pool

LETTERS_POOL = get_letters_pool()
```

Reply:
this way, if the letter scores or availability change, then you only need to change the original dictionary!